R/object_usage_linter.R

```diff
@@ -160,13 +160,16 @@ extract_glued_symbols <- function(expr) {
   }
   glued_symbols <- new.env(parent = emptyenv())
 
+  unexpected_error <- function(cond) {
+    stop("Unexpected failure to parse glue call, please report: ", conditionMessage(cond))
```

Comment:
Can we add a test to make sure that we get this expected error message?

Reply:
no -- I don't know how to trigger the error. it should be nocov but covr doesn't separate out tryCatch branches